docs/source-pytorch/common/trainer.rst

```diff
@@ -1064,15 +1074,15 @@ With :func:`torch.inference_mode` disabled, you can enable the grad of your mode
 
 .. code-block:: python
 
-    class LitModel(LightningModule):
+    class LightningTransformer(pl.LightningModule):
```

Comment:
Since you already import a class with this name in the snippet above, why not keep this other class with a different name?

Reply:
I just wanted to be consistent in the model names used throughout.